### ISSUE TITLE
VAMC Operating status: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16998]
## Description
VAMC Operating Status pages need the following updates to components:

Example pages:

- [Boston VAMC Operating status](https://www.va.gov/boston-health-care/operating-status/)
- [Minneapolis VAMC Operating status](https://www.va.gov/minneapolis-health-care/operating-status/)

### Elements that are not using components that should be (Backlog)
1. Link
   - Facility names should be using the component
   - Any other links should be using the component if possible, with the understanding that CMS content has limitations
2. ~Telephone~
   - ~[The component](https://design.va.gov/components/telephone) is not being used anywhere but should be if possible, with the understanding that CMS content has limitations~
   - ~Confirm exts are parsed correctly in the component~
3. Alert - Slim
   - Facility Statuses are not using a component but they should use the [v3 slim alert](https://design.va.gov/components/alert#examples---slim-alert---v3)
   - UPDATE 5/2: [From a slack conversation](https://dsva.slack.com/archives/C0FQSS30V/p1714681783595959?thread_ts=1714673084.027969&cid=C0FQSS30V), we are updating the "Limited services and hours" status to use the yellow `warning` alert status. This will make it align with the alerts status used on the Facility locator and the Facility detail pages. (Currently we use yellow on these pages but blue here.)
4. ~Back to top~
   - ~Does this page meet the criteria guidelines for the [Back to top component](https://design.va.gov/components/back-to-top)?~

## Acceptance Criteria
- [ ] Link component has been added
- [ ] Alert - Slim has been added
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I would estimate this at a **5** due to the large number of new components being added in, and we need to make sure they all work together on the page as expected.

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by laflannery** [comment]
Removed Slim alert because it was moved into #17855

**Comment by eselkin** [comment]
I estimate a 3; while `Emergency information` is Drupal content, `Local emergency resources` can be updated because it iterates over links and puts them in anchor tags. Additionally, the links to facilities can be updated. It's not too hard, but being cautious about the emergency links is good.

**Comment by randimays** [comment]
Adding a note here: this page does not meet the criteria for a Back to Top component per [the DS guidelines](https://design.va.gov/components/back-to-top#when-to-use-back-to-top) as it is dynamic content.

<img width="606" alt="Screenshot 2024-05-02 at 2 34 46 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/b5eab2d6-5a5e-4936-8686-98e8bb0c402f">

**Comment by laflannery** [comment]
Updating the ticket description to:
- un-strikeout the alert component updates, as it's getting done here and not in the icon ticket
- also added a note about updating the Limited services status to use the warning state to be in alignment with the rest of this status sitewide

**Comment by laflannery** [comment]
@randimays Confirming that I don't think we need to add the Back to top component here. The pages don't tend to vary greatly in length but also aren't super long. I think we can revisit this when the component is updated with the page height detection

**Comment by randimays** [comment]
In working on a different ticket, I discovered an issue with converting the facility name links into `<va-link>`. Some of the facility names have a double quote, such as `Hershel "Woody" VA Medical Center`. Passing this name into a `<va-link>` breaks the component and only renders everything up to the first double quote. The code for this ticket is already in production and is broken in these instances. I'm working to figure out what the desired solution is. Slack thread: https://dsva.slack.com/archives/C06NMUK12LR/p1715026428947379

If we need some time to figure out how to handle these instances, I will revert the PR and we can come back to this later.
